bundles/org.openhab.ui/web/src/components/widgets/widget-mixin.js

```diff
@@ -20,13 +20,18 @@ expr.jsep.plugins.register(jsepRegex, jsepArrow, jsepObject, jsepTemplate)
 
 expr.addUnaryOp('@', (itemName) => {
   if (itemName === undefined) return undefined
-  const itemState = store.getters.trackedItems[itemName]
-  return itemState.displayState || itemState.state
+  const item = store.getters.trackedItems[itemName]
+  return item.displayState || item.state
 })
 expr.addUnaryOp('@@', (itemName) => {
   if (itemName === undefined) return undefined
   return store.getters.trackedItems[itemName].state
 })
+expr.addUnaryOp('#', (itemName) => {
+  if (itemName === undefined) return undefined
+  const item = store.getters.trackedItems[itemName]
+  return item.numericState || item.state
```

Comment:
IMO it doesn't make sense to fallback to the string state.

Reply:
agreed totally!

Reply:
how about `|| Number.parseFloat(item.state)` ?

Reply:
As it is a shortcut for `numericState`, I think it should behave the same; so no fallback to parseFloat.

Reply:
OK in that case I'll leave the last commit as is (no fallback)

Reply:
Very minor thing: Could you please remove the item const and make it a single liner?

Reply:
D'oh! Thanks for noticing that!